packages/widget/src/ui/Widget.tsx

```diff
@@ -294,7 +294,10 @@ export const SwapWidgetUI = ({
                 }
               }}
             >
-              <div key={accountStateKey} className="animate-slide-up-and-fade">
+              <div
+                key={accountStateKey}
+                className="animate-slide-up-and-fade text-white"
+              >
```

Comment:
`div/p/span` are now defaulted to color: black; so this needs to explicitly have text-white (does not inherit it from parent anymore)